client/viame-web-common/components/AttributesPanel.vue

```diff
@@ -257,21 +259,26 @@ export default defineComponent({
         :lock-types="lockTypes"
         @seek="$emit('track-seek', $event)"
       />
-      <v-subheader class="border-highlight">
-        <v-row>
+
+      <!-- CONFIDENCE PAIRS -->
+      <div class="border-highlight">
+        <v-row
+          no-gutters
+          class="align-center"
+        >
```

Comment:
I made this exact change in all three headings.  

* subheader was too bloated
* row horizontal alignment was off.